src/view/drag-drop-context/use-startup-validation.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import React, { useEffect } from 'react';
+import { peerDependencies } from '../../../package.json';
+import checkReactVersion from './check-react-version';
+import checkDoctype from './check-doctype';
+
+export default function useStartupValidation() {
+  // Only need to run these when mounting
+  useEffect(() => {
+    if (process.env.NODE_ENV === 'production') {
+      return;
+    }
+
+    checkReactVersion(peerDependencies.react, React.version);
```

Comment:
I would wrap this check with condition. It looks like early returns does not eliminated well with webpack uglify pair.

Reply:
Thanks

Reply:
TODO: update